Demo/Demo/Models/CreateOrderParams.swift

```diff
@@ -30,15 +62,17 @@ struct VaultCard: Encodable {
 struct Attributes: Encodable {
 
     let vault: Vault
-    let customer: CardVaultCustomer?
+    let customer: VaultCustomer?
```

Comment:
I wonder if we can just call this `Customer` instead of `VaultCustomer`.

Reply:
Addressed here: 62480f60df4f9705c4092f02d9a983298e08833e